.azure/pipelines/devBuilds.yml

```diff
@@ -3,11 +3,7 @@
 #
 
 # Configure which branches trigger builds
-trigger:
-  batch: true
-  branches:
-    include:
-    - master
+trigger: none
```

Comment:
Does this mean there are no triggers at all?

Reply:
Yes, for now. Till I get this working properly.